packages/docs/stories/components/side-nav.stories.tsx

```diff
@@ -82,24 +109,25 @@ export const Default: Story = {
     },
   },
   render: (args: unknown) => html`
-  <syn-header>Side Navigation</syn-header>
-  <main style="position: relative; height: 500px; background-color: var(--syn-color-neutral-200);" class="side-nav-default">
-    ${generateTemplate({ args })}
-  </main> 
-  <script type="module">
-    const mainContents = document.querySelectorAll('.side-nav-default');
+    <syn-header>Side Navigation</syn-header>
+    <main style="position: relative; height: 500px; background-color: var(--syn-color-neutral-200);" class="side-nav-default">
+      ${generateTemplate({ args })}
+    </main> 
+    <script type="module">
+      const mainContents = document.querySelectorAll('.side-nav-default');
 
-    Array.from(mainContents).forEach((mainContent, index) => {
-      const selector = 'story-loaded-'.concat(index);
-      if(!mainContent.classList.contains(selector)) {
-        const header = mainContent.previousElementSibling;
-        const sideNav = mainContent.querySelector('syn-side-nav');
-        header.connectSideNavigation(sideNav);
-        mainContent.classList.add(selector);
-       }
-    });
-  </script>
-`,
+      Array.from(mainContents).forEach((mainContent, index) => {
+        const selector = 'story-loaded-'.concat(index);
+        if(!mainContent.classList.contains(selector)) {
+          const header = mainContent.previousElementSibling;
+          const sideNav = mainContent.querySelector('syn-side-nav');
+          header.connectSideNavigation(sideNav);
+          mainContent.classList.add(selector);
+        }
+      });
+    </script>
+    ${createDummyActiveNavItemListener()}
```

Comment:
Really only tested it on the docs page instead of the single stories ^^. It is built other ways as the other stories, so I was happy there was just a single entry to do and it worked. Should have looked better, its fixed now.